apis/HdmiCecSource/IHdmiCecSource.h

```diff
@@ -0,0 +1,171 @@
+            // @param vendorId: Vendor ID of the HDMI CEC source
+            // @param success: Is the operation successful or not
+            virtual Core::hresult GetVendorId(string &vendorid /* @out */, bool &success /* @out */) = 0;
+
+            // @brief Performs the OTP action
+            // @text performOTPAction
+            // @param success: Is the operation successful or not
+            virtual Core::hresult PerformOTPAction(bool &success /* @out */) = 0;
+
+            // @brief Sends a key press event to the HDMI CEC device.
+            // @text sendKeyPressEvent
+            // @param logicalAddress: Logical address of the device
+            // @param keyCode: Key code of the key press event
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SendKeyPressEvent(const uint32_t &logicalAddress /* @in */, const uint32_t &keyCode /* @in */, bool &success /* @out */) = 0;
+
+            // @brief Sends a standby message to another CEC device
+            // @text sendStandbyMessage
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SendStandbyMessage(bool &success /* @out */) = 0;
+
+            // @brief Sets the status of the HDMI CEC source
+            // @text setEnabled
+            // @param enabled: Is the HDMI CEC source enabled or not
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SetEnabled(const bool &enabled /* @in */, bool &success /* @out */) = 0;
+
+            // @brief Sets the OSD name of the HDMI CEC source
+            // @text setOSDName
+            // @param osdName: OSD name of the HDMI CEC source
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SetOSDName(const string &name /* @in */, bool &success /* @out */) = 0;
+
+            // @brief Sets the OTP enabled status of the HDMI CEC source
+            // @text setOTPEnabled
+            // @param enabled: Is the OTP enabled or not
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SetOTPEnabled(const bool &enabled /* @in */, bool &success /* @out */) = 0;
+
+            // @brief Sets the vendor ID of the HDMI CEC source
+            // @text setVendorId
+            // @param vendorId: Vendor ID of the HDMI CEC source
+            // @param success: Is the operation successful or not
+            virtual Core::hresult SetVendorId(const string &vendorid /* @in */, bool &success /* @out */) = 0;
+
+            // @brief Gets the list of devices connected to the HDMI CEC source
+            // @text getDeviceList
+            // @param deviceList: List of devices connected to the HDMI CEC source
+            // @param success: Is the operation successful or not
+            virtual Core::hresult GetDeviceList(IHdmiCecSourceDeviceListIterator*& deviceList /* @out */, bool &success /* @out */) = 0;
```

Comment:
"numberofdevices" parameter is missing as per doc: 
https://rdkcentral.github.io/rdkservices/#/api/HdmiCecSourcePlugin?id=response-1